application/common/manifest_handlers/tizen_metadata_handler.cc

```diff
@@ -66,9 +66,9 @@ bool TizenMetaDataHandler::Parse(scoped_refptr<ApplicationData> application,
   const Manifest* manifest = application->GetManifest();
   DCHECK(manifest);
 
-  base::Value* metadata_value = NULL;
+  base::Value* metadata_value = nullptr;
   if (!manifest->Get(keys::kTizenMetaDataKey, &metadata_value)) {
-    *error = base::ASCIIToUTF16("Failed to get value of tizen metaData");
+    return true;
```

Comment:
Methods Get() and HasPath() calls CanAccessPath() internally so I made choise just to call one.
Setting error is not needed.